### PR TITLE
Motive -> Painting Motive

### PR DESCRIPTION
**Pull request by Earthcomputer**
A less controversial version of #1076. Renaming `Registry.MOTIVE` to `Registry.PAINTING_MOTIVE` and adding Javadocs seemed to have the benefits of both sides of the argument in the discussion on that PR, so that's what this PR does.

### CODE REVIEW

File: mappings/net/minecraft/network/packet/s2c/play/PaintingSpawnS2CPacket.mapping [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 	FIELD field_12008 pos Lnet/minecraft/class_2338;
 	FIELD field_12009 uuid Ljava/util/UUID;
-	FIELD field_12010 motive I
+	FIELD field_12010 motiveId I
 	FIELD field_12011 facing Lnet/minecraft/class_2350;
 	FIELD field_12012 id I
 	METHOD <init> (Lnet/minecraft/class_1534;)V
 		ARG 1 entity
-	METHOD method_11221 getMotive ()Lnet/minecraft/class_1535;
+	METHOD method_11221 getMotif ()Lnet/minecraft/class_1535;
```
Comment (Juuxel) [inline mappings/net/minecraft/network/packet/s2c/play/PaintingSpawnS2CPacket.mapping]: Is this supposed to still be `motif`?

Comment (Earthcomputer) [reply mappings/net/minecraft/network/packet/s2c/play/PaintingSpawnS2CPacket.mapping]: Whoops, fixed

**Review by LambdAurora — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Review by Prospector — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]
Unlike some other registry renaming proposals, I agree with this one. Adding a clarifying prefix while not fully renaming something registered is a good way to go about it.

**Review by Runemoro — APPROVED** [review]

**Comment by Sollace** [comment]
You didn't fix the typo though

**Comment by Sollace** [comment]
I maintain that Painting Motive is still wrong because a motive is not a motif. This is _basic_ stuff, people.

**Comment by Earthcomputer** [comment]
I agree, and I would have preferred your PR to be merged. However, the reality was that your PR was not going to be merged, so a compromise had to be met. If it's between `motive` and `painting_motive`, the latter is 1000 times better.

**Comment by grondag** [comment]
What the hell, people? Words mean things. Painting *motive* makes zero damn sense. 

It’s like someone calls me Fred, and then when I point out that my name isn’t Fred they start calling me Mister Fred. And when I complain again, they say at least they made it more clear.

**Comment by Earthcomputer** [comment]
See the arguments in the other PR. "Motive" is an alternative, albeit much rarer, spelling of "motif". It isn't incorrect, just confusing, so needed clarification.

**Comment by grondag** [comment]
Ok. Reading the other PR I see that *somehow* motive is now a synonym for motif, which just further confirms that the universe is an uncaring void, punctuated with random crap.

I’m going to go find a cane now, and then stand on my lawn and shake it at youngsters passing by, telling them that in my day we cared about proper spelling of words, and didn’t just hack into online dictionaries to change words that we kept getting wrong. 

And don’t get me started about “comprise” ...

**Comment by Sollace** [comment]
> 
> 
> See the arguments in the other PR. "Motive" is an alternative, albeit much rarer, spelling of "motif". It isn't incorrect, just confusing, so needed clarification.

They're literally completely different words with different meanings. "Motive"is in no way, meaning, or form, and alternative spelling of "motif"

**Comment by Sollace** [comment]
Side note: I asked in a Discord server filled with writers who spend their time writing and answering these sorts of questions, and who are generally right when it comes to these things.
image

**Comment by Sollace** [comment]
Oops. I missed one.
image

**Comment by liach** [comment]
Seriously @Sollace Time to stop! Don't fuck up with tons of trivial refactors!
image

**Pull request merged**